vita3k/modules/SceAvPlayer/SceAvPlayer.cpp

```diff
@@ -219,11 +220,41 @@ static Ptr<uint8_t> get_buffer(const PlayerPtr &player, MediaType media_type,
     return buffer;
 }
 
+struct ThreadParamsEx {
+    KernelState *kernel = nullptr;
+    ThreadState *thread = nullptr;
+    SceUID thid = 0;
+    Address callback_address = 0;
+    uint wait_time = 0;
```

Comment:
Reason? This seems like race condition prevention hack

Reply:
like i said here, is for test only and for any other dev can look and try debug on this :) 
anhd for mysterious reason, if i no add delay that no works, but i can using 10 that works too